arches_lingo/src/arches_lingo/pages/ConceptPage.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import { inject } from "vue";
+
+import { displayedRowKey } from "@/arches_lingo/constants.ts";
+
+import type { DisplayedRowRefAndSetter } from "@/arches_lingo/types";
+
+const { displayedRow } = inject(
+    displayedRowKey,
+) as unknown as DisplayedRowRefAndSetter;
```

Comment:
`as unknown as DisplayedRowRefAndSetter`?

Reply:
Yeah, the tooling is aggressively inferring the result of `inject()` as the first element (the ref) not the element I'm actually unpacking: the setter. Unpacking both elements didn't help either, just caused more eslint warnings.

If I find some energy after Christmas I'll try cooking up a repro for vue-tsc, maybe they'll let me know of something silly I'm overlooking!

Reply:
See the advice to use unknown as the escape hatch:
```ts
Conversion of type 'Ref<Concept | null, Concept | null> | undefined' to type 'DisplayedRowRefAndSetter' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'Ref<Concept | null, Concept | null>' is missing the following properties from type 'DisplayedRowRefAndSetter': displayedRow, setDisplayedRowts-plugin(2352)
```

Reply:
++  --thanks for looking into it 👍